src/azure-openapi-linter/rules/ArraySchemaMustHaveItems.ts

```diff
@@ -12,19 +12,14 @@ rules.push({
   name: ArraySchemaMustHaveItems,
   severity: "warning",
   category: "SDKViolation",
-  mergeState: MergeStates.individual,
+  mergeState: MergeStates.composed,
```

Comment:
Either should do really since this works irrespective of whether the spec is in merged state or not.